lab-kyle/index.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
```

Comment:
This is a good start but unfortunately this method does not guarantee that the files will be logged in order.